Demo/AppDelegate.swift

```diff
@@ -29,6 +32,46 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 }
 
+extension AppDelegate {
+    enum FontType: String {
+        case light = "FINNTypeWebStrippet-Light"
+        case medium = "FINNTypeWebStrippet-Medium"
+        case regular = "FINNTypeWebStrippet-Regular"
+        case bold = "FINNTypeWebStrippet-Bold"
+    }
+
+    func registerFonts() {
```

Comment:
Does it mean we'll have to do the same kind of font registration in the app? Perhaps we could make FINN fonts and colors as default in `Bootstrap`? Or add some sort of helper `registerFINNFonts` function in `FontBook` extension in `FinniversKit` if we don't want to move our custom fonts to `Bootstrap`.

Reply:
> Does it mean we'll have to do the same kind of font registration in the app?

Yes, the idea is that the hosting app injects custom fonts, if you don't do that it will use the default fonts. If we embed the FINN fonts in Bootstrap it's most likely that we'll see a bunch of apps using the FINN font and we definitely don't want that. We'll most likely remove the colors and make the default colors the iOS colors but that can wait into a future PR.